cli/inspect/inspector_test.go

```diff
@@ -7,9 +7,14 @@ import (
 	"strings"
 	"testing"
 	"text/template"
+
+	"github.com/alibaba/pouch/pkg/utils/templates"
+	"github.com/google/go-cmp/cmp"
 )
 
 func TestNewTemplateInspector(t *testing.T) {
+	// Prepare test data
+	tmpl1, _ := template.New("template1").Parse("{{.ID}}")
```

Comment:
it will be better if we can give a meaningful name for 'template1' and 'tmpl1', in addition, I haven't also see variable name like 'template2' and 'tmpl2'.

Reply:
Thank you for your review. How about names it 'idTmpl' and 'idTemplate'? I will change it in the latest commit.

Reply:
anyway, it's okay as long as the name is meaningful